script.js

```diff
@@ -1,4 +1,264 @@
+// All the face cards have a value of 10. The functions are exclusively for BlackJack deck.
+var myImageWin =
+  '<img src = "https://c.tenor.com/Sk82dtcF6bUAAAAC/winner-winner-chicken-dinner.gif"/>';
+var myImageLose =
```

Comment:
I like how you saved the image links as variables so that it's clear when you use them